hack/operatorhub/config.yaml

```diff
@@ -1,4 +1,4 @@
-newVersion: 2.8.0-SNAPSHOT
+newVersion: 2.7.0
```

Comment:
Was this intentional? Seems outside of the scope of this PR.

Reply:
It was just for testing, reverting.